### PR TITLE
Update korean.json

### PR DESCRIPTION
**Pull request by Philippians-4-13**
Is this how I have to do it?

### CODE REVIEW

File: server/data/words/korean.json [hunk 1]
```diff
@@ -1,2022 +1,2022 @@
...
     "translation": "anniversary"
   },
   {
     "word": "알리다",
-    "romanization": "",
+    "romanization": "allida",
     "translation": "announce"
   },
   {
     "word": "일년생 식물",
```
Comment (Philippians-4-13) [inline server/data/words/korean.json:623]: 일년생 식물 sounds a bit off, to be honest

**Comment by Philippians-4-13** [comment]
Perfect, thank you very much @jayehernandez !

**Comment by Philippians-4-13** [comment]
Cool! I'm still on the job and I've seen some verbs in form of nouns, so I fix it by changing it to the dictionary form. Is that ok?

**Comment by jayehernandez** [comment]
> Cool! I'm still on the job and I've seen some verbs in form of nouns, so I fix it by changing it to the dictionary form. Is that ok?

Yes this is okay!

**Comment by jayehernandez** [comment]
Do let me know when you've finished changing it to dictionary form. Thank you!

**Comment by jayehernandez** [comment]
> @jayehernandez I've just finished with all. The PR is ready to be merged.

Thank you so much for working on this!! 🎉 I'll be including it in the next release.

> By the way, I would like to add some quotes from Jorge Luis Borges and Yi Sun Shin.

For this one, @Philippians-4-13, you can create a new Pull Request! :)

**Comment by jayehernandez** [comment]
@all-contributors please add @Philippians-4-13 for translation

**Pull request merged**